src/dispatcher/actions.ts

```diff
@@ -69,4 +69,14 @@ export enum Action {
      * Opens the user menu (previously known as the top left menu). No additional payload information required.
      */
     ToggleUserMenu = "toggle_user_menu",
+
+    /**
+     * Sets the apps root font size. Should be used with UpdateFontSizePayload
+     */
+    UpdateFontSize = "update_font_size",
```

Comment:
Should this font size part be in a different PR...?

Reply:
This doesn't change the behaviour of font sizing in any way. I just thought I would opportunistically move this to type-checked ts while I was in the area.